nx-X11/include/Xos_r.h

```diff
@@ -328,24 +320,17 @@ static __inline__ void _Xpw_copyPasswd(_Xgetpwparams p)
   (p).pwp )
 
 #elif !defined(_POSIX_THREAD_SAFE_FUNCTIONS) && !defined(__APPLE__) && !defined(__DARWIN__)
-/* SVR4 threads, AIX 4.2.0 and earlier and OSF/1 3.2 and earlier pthreads */
+/* SVR4 threads and OSF/1 3.2 and earlier pthreads */
```

Comment:
Yes, And some more, actually. I will push various commits with "rebase <platform>", so reviewing should be easier. Before merging, I need to flatten out those commits, though.